test/validation/api/event/event.c

```diff
@@ -203,6 +304,186 @@ static void event_test_pool_info_ctx(void)
 	}
 }
 
+static void event_test_user_area_and_flag_ctx(void)
+{
+	event_type_ctx_t *ctx = &g_type_ctx;
+	odp_event_t ev;
+	odp_event_type_t ev_t;
+	int i, j, flag;
+	void *user_area;
+
+	for (i = 0; i < MAX_EVENT_TYPES; i++) {
+		for (j = 0; j < EVENT_BURST; j++) {
+			ev = ctx->events[i][j];
+			user_area = odp_event_user_area(ev);
+			CU_ASSERT(user_area != NULL);
+
+			flag = 1;
+			odp_event_user_flag_set(ev, flag);
+			CU_ASSERT(odp_event_user_area_and_flag(ev, &flag) != NULL);
+
+			ev_t = ctx->event_types[i];
+			if (ev_t == ODP_EVENT_PACKET)
+				CU_ASSERT(flag > 0);
+		}
+	}
+}
+
+static void event_test_misc_ctx(void)
+{
+	event_type_ctx_t *ctx = &g_type_ctx;
+	odp_event_t ev;
+	int i, j;
+
+	for (i = 0; i < MAX_EVENT_TYPES; i++) {
+		for (j = 0; j < EVENT_BURST; j++) {
+			ev = ctx->events[i][j];
+
+			CU_ASSERT(odp_event_to_u64(ev) != 0);
```

Comment:
API says nothing about zero value, so this should not be tested. One could check that individual events return different u64 values.